src/SFA.DAS.AANHub.Data/Repositories/CalendarEventsReadRepository.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Microsoft.EntityFrameworkCore;
+using SFA.DAS.AANHub.Domain.Entities;
+using SFA.DAS.AANHub.Domain.Interfaces.Repositories;
+
+namespace SFA.DAS.AANHub.Data.Repositories;
+internal class CalendarEventsReadRepository : ICalendarEventsReadRepository
```

Comment:
you can add a '[ExcludeFromCodeCoverage]' to this

Reply:
Attribute added